src/main/java/io/gatling/mojo/EnterpriseDeployMojo.java

```diff
@@ -58,7 +59,7 @@ public void execute() throws MojoFailureException {
   private File getDeploymentFile() {
     File baseDir = mavenProject.getBasedir();
     if (customPackageFilename == null) {
-      return DeploymentConfiguration.fromBaseDirectory(baseDir, customPackageFilename);
+      return DeploymentConfiguration.fromBaseDirectory(baseDir, null);
```

Comment:
Why?

Reply:
I think it's a bit more readable/makes it a bit more obvious what's going on, since we're in the branch where we know it's `null`. Doesn't make a huge difference to me one way or the other, though. 🤷